examples/window/window_settings.rs

```diff
@@ -8,6 +10,12 @@ fn main() {
             width: 500.,
             height: 300.,
             vsync: true,
+            #[cfg(any(target_os = "windows", target_os = "linux"))]
+            icon: Some(Icon::from_rgba(
+                include_bytes!("bevy_icon.rgba").to_vec(),
```

Comment:
I suppose if this used `Cow<'static, [u8]>` we could avoid the `to_vec`, in case another backend supported `&'static [u8]` icons for example

That being said, requiring the user to create the Vec to match the winit API is also reasonable.